config.yaml

```diff
@@ -1,8 +1,8 @@
 default:
  log_file: "/var/log/host-os/builds.log"
  packages_repos_target_path: "/var/lib/host-os/repositories"
- build_versions_repository_url: "https://github.com/open-power-host-os/versions.git"
- build_version: 'master'
+ packages_metadata_repo_url: "https://github.com/open-power-host-os/versions.git"
```

Comment:
Since this is exposed to the user and a common option to use, I'd prefer to use a more relatable name, like `packages_versions_repo_url`.